docs/pages/Accessibility.js

```diff
@@ -249,4 +250,6 @@ card(
   </MainSection>,
 );
 
-export default cards;
+export default function Accessibility(): Node {
+  return <CardPage cards={cards} page="Accessibility" />;
```

Comment:
This syntax is a lot nicer. Much more readable and obvious how these cards are being displayed.